dev/tasks/python-wheels/github.linux.yml

```diff
@@ -49,6 +71,7 @@ jobs:
           archery docker run python-wheel-manylinux-test-imports
           archery docker run python-wheel-manylinux-test-unittests
 
+      {% if arch == "amd64" %}
```

Comment:
Can we enable this on arm64 too?

Reply:
I did not enable these in arm64 because Travis job did not seem having this, but I will try it out.